src/NServiceBus.AcceptanceTests/ApprovalFiles/When_enpoint_is_warmed_up.Make_sure_things_are_in_DI.approved.txt

```diff
@@ -0,0 +1,19 @@
+[NServiceBus.AcceptanceTesting.ScenarioContext, NServiceBus.AcceptanceTesting.ScenarioContext - SingleInstance (WasResolved: False)]
+[NServiceBus.CriticalError, NServiceBus.CriticalError - SingleInstance (WasResolved: False)]
+[NServiceBus.DefaultScheduler, NServiceBus.DefaultScheduler - SingleInstance (WasResolved: False)]
+[NServiceBus.Hosting.HostInformation, NServiceBus.Hosting.HostInformation - SingleInstance (WasResolved: True)]
```

Comment:
I wonder if it makes sense to have two approval files. One for the resolved ones and one for the registered only ones. Ultimately we want the core to not resolve its own stuff but for backward compatibility, we need to keep the only registered ones probably because they could be resolved by a downstream